torch/_tensor_docs.py

```diff
@@ -3619,11 +3684,33 @@ def callable(a, b) -> number
 See :func:`torch.topk`
 """)
 
+add_docstr_all('to_dense',
+               r"""
+to_dense() -> Tensor
```

Comment:
Should this be "to_strided()"?

Reply:
I would agree. This API change should be discussed in a separate issue/PR/RFC.

Reply:
Created https://github.com/pytorch/pytorch/issues/46165